package.json

```diff
@@ -84,6 +84,7 @@
     "http-server": "^13.1.0",
     "husky": "^7.0.4",
     "lint-staged": "^10.5.4",
+    "lit-html": "^2.0.0",
```

Comment:
why is that needed? 🤔

Reply:
Other wise lit-html@1.0 becomes the “default” due to Haunted, Lion and Rocket et al and the types get built as `lit/nodes_modules/lit-html` which isn’t guaranteed to be where that dependency is in other projects. This way it gets built at `lit-html` and can be resolved more naturally by consuming projects.